### PR TITLE
KEP-1796 bzapi-cli help contains typos

### PR DESCRIPTION
**Pull request by ebruck**
KEP-1794 bzapi build fails with -DUSE_JSONCPP_SO=ON

### CODE REVIEW

File: bzapi-cli/main.cpp [hunk 1]
```diff
@@ -559,82 +559,82 @@ main(int argc, const char* argv[])
             ("uuid,u",   po::value<std::string>(), "database uuid");
 
         // some help as subparsers don't exists ("easily") in boost::program_options...
-        po::options_description status("status (Swarm status)");
```
Comment (rnistuk) [inline bzapi-cli/main.cpp:562]: Cool replacing magic strings with constants.

File: bzapi-cli/main.cpp [hunk 2]
```diff
@@ -559,82 +559,82 @@ main(int argc, const char* argv[])
...
             (HAS_KEY.c_str(), po::value<std::string>(), "key to check");
         description.add(has);
 
-        po::options_description ttl("ttl (Get ttl for a key within a db by uuid)");
+        po::options_description ttl(TTL_CMD + " (Get ttl for a key within a db by uuid)");
         ttl.add_options()
             (TTL_KEY.c_str(), po::value<std::string>(), "ttl for a key");
         description.add(ttl);
 
-        po::options_description persist("persis (Remove expiration for a key within a database by uuid)");
```
Comment (rnistuk) [inline bzapi-cli/main.cpp:626]: Is this the spelling mistake?

File: bzapi-cli/CMakeLists.txt [hunk 3]
```diff
@@ -1,4 +1,10 @@
 add_executable(bzapi-cli main.cpp)
-add_dependencies(bzapi-cli boost jsoncpp)
+
```
Comment (rnistuk) [inline bzapi-cli/CMakeLists.txt:2]: ok, only add jsoncpp if required

**Review by rnistuk — APPROVED** [review]
I think it look good.

File: bzapi-cli/main.cpp (hunk 2 above)
Comment (ebruck) [reply bzapi-cli/main.cpp]: yeah I'll fix that
thx!

**Pull request merged**